docs/Release_notes.md

```diff
@@ -1,6 +1,35 @@
 CARMA Platform Release Notes
 ----------------------------
 
+Version Testarossa, released December 23rd, 2019
+--------------------------------------------------------
+
+**Summary:**
+CARMAPlatform release version Testarossa includes the following four major updates. An initial converter has been developed to convert OpenDrive maps to LaneLet2's OSM format. The converter currently only covers lane geometry only. Autoware v1.13 pre-release has been integrated with CARMAPlatform. A guidance plan delegator has been developed to notify strategic plugins that the arbitrator has selected their maneuver plan, and request the correstponding trajectory plan for said plugin. Finally, a guidance re-engage capability has been added to the platform, which will allow for multiple runs to be made without restarting the software.  
+
+Enhancements in this release:  
+- Issue 416: Update Docker to use Autoware 1.13 build
```

Comment:
This should still say Autoware 1.12 build. the one issue below is the one for the pre-release of v1.13. We have not upgraded to the official release of 1.13, so can't say that yet.